src/clojure/duratom/not_found/redis.clj

```diff
@@ -0,0 +1,26 @@
+(ns duratom.not-found.redis)
+
+(defmacro wcar [conn-opts & args]
+  `(let ~conn-opts
```

Comment:
I don't think you need `conn-opts`, nor `let`, do you?

Reply:
Nope. Fixed.